### DISCUSSION TITLE
Roaster ID

### DISCUSSION THREAD

**Discussion by Abjudge** [artisan-roaster-scope/artisan#1621] [Q&A] [answered]
After exporting data as a JSON file, is there any identifier, like an ID, that specifies which roaster was used? For instance, if I have multiple roasters, can I determine from the JSON file which one was used for a particular export?

**Comment by MAKOMO** [comment, marked as answer]
No. Machines are only identified by their name string that can be set under menu Roast >> Properties,  last tab "Setup", attribute "Model".
